### DISCUSSION TITLE
HEIMAN HS1CA-E Problem with the rights values

### DISCUSSION THREAD

**Discussion by avdgaag1984** [Koenkk/zigbee2mqtt#11472] [Devices]
Good evening,

Since today I have a Carbon monoxide detector from Heiman Type HS1CA-E

I managed to add this 1x successfully, all values ​​were read neatly, but had to remove it 1x because of. wrong. Now I have re-added it several times but only the signal strength is displayed. Anyone have an idea what I'm doing wrong?

![image](https://user-images.githubusercontent.com/91500867/154568981-31a2401a-6425-48df-bf87-702510fd8ffe.png)

Log from zigbee2mqtt: 
Zigbee2MQTT:info  2022-02-17 21:34:58: MQTT publish: topic 'homeassistant/sensor/0x000d6f0014f82c9f/battery/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x000d6f0014f82c9f"],"manufacturer":"HEIMAN","model":"Smart carbon monoxide sensor (HS1CA-E)","name":"CO-Melder","sw_version":"1.1.1"},"device_class":"battery","enabled_by_default":true,"entity_category":"diagnostic","json_attributes_topic":"zigbee2mqtt/CO-Melder","name":"CO-Melder battery","state_class":"measurement","state_topic":"zigbee2mqtt/CO-Melder","unique_id":"0x000d6f0014f82c9f_battery_zigbee2mqtt","unit_of_measurement":"%","value_template":"{{ value_json.battery }}"}'

**Comment by GegoSK** [comment]
Finally someone with same problem! I have problem with devices Heiman HS1CA-E and Heiman HS3SA. States was lost, when I updated Home Assistant from 2021.12.10 to 2022.2.6. I tried to repair many times, addon even HA restarted many times. Sometime it shows "null", sometime "N/A". LQI and battery state are available, 255 and 100% in my case. I do not know, what to do.

**Reply by GegoSK** [reply]
After few weeks, Heiman HS1CA-E was paired with right values. One day I woke up and it was paired. But HS3SA is still not working. But this morning I found in a log:

` 2022-05-07 09:27:46Failed to configure 'Dymový senzor', attempt 1 (Error: Bind 0x000d6f0016194184/1 genPowerCfg from '0x00212effff05cbb2/1' failed (waiting for response TIMEOUT) at DeconzAdapter.bind (/app/node_modules/zigbee-herdsman/src/adapter/deconz/adapter/deconzAdapter.ts:761:19) at Endpoint.bind (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:567:13) at Object.bind (/app/node_modules/zigbee-herdsman-converters/lib/reporting.js:40:9) at Object.configure (/app/node_modules/zigbee-herdsman-converters/devices/heiman.js:97:13) at Configure.configure (/app/lib/extension/configure.ts:115:13) at Configure.onMQTTMessage (/app/lib/extension/configure.ts:55:21))`

I tried to burn a paper under sensor and no alarm.

**Reply by GegoSK** [reply]
[avdgaag1984](https://github.com/avdgaag1984) One year after. Did you managed to repaired it? I have still same problem. Sensor most probably will go to the trash.
